lore/util.py

```diff
@@ -133,6 +133,9 @@ def flush(self):
 }
 logger.setLevel(log_levels.get(env.NAME, logging.INFO))
 
+_sql_alchemy_logger = logging.getLogger('sqlalchemy')
+if _sql_alchemy_logger is not None:
+  _sql_alchemy_logger.setLevel(os.environ.get('LORE_SQLALCHEMY_LOGGING_LEVEL', 'INFO'))
```

Comment:
I believe you need 4 spaces to match the rest of the file's indentation or Python will be unhappy.
I also switched it to `getenv`, which does the same thing so you can change it back if you'd prefer. Just a bit shorter.
```suggestion
    _sql_alchemy_logger.setLevel(os.getenv('LORE_SQLALCHEMY_LOGGING_LEVEL', 'INFO'))
```